contracts/bonding/IBondingManager.sol

```diff
@@ -24,6 +24,9 @@ contract IBondingManager {
     // event Bond(address indexed delegate, address indexed delegator);
     // event Unbond(address indexed delegate, address indexed delegator);
     // event WithdrawStake(address indexed delegator);
+    // event TranscoderUpdate(address indexed transcoder, uint256 pendingRewardCut, uint256 pendingFeeShare, bool registered);
```

Comment:
Let's update this deprecated event signature to reflect the current [event signature](https://github.com/livepeer/protocol/blob/master/contracts/bonding/IBondingManager.sol#L9) on `master`

Reply:
fixed in 36f8765